spark/v3.5/spark/src/test/java/org/apache/iceberg/spark/source/TestDataSourceOptions.java

```diff
@@ -297,7 +299,7 @@ public void testIncrementalScanOptions() throws IOException {
             .orderBy("id")
             .as(Encoders.bean(SimpleRecord.class))
             .collectAsList();
-    Assert.assertEquals("Records should match", expectedRecords.subList(1, 4), result);
+    assertThat(expectedRecords.subList(1, 4)).as("Records should match").isEqualTo(result);
```

Comment:
this should be the other way around: `expectedRecords.subList(1, 4)` should go into `isEqualTo()` as that's what's being expected.

It looks like the assertions are the other way around in a bunch of places. Could you please fix those?

Reply:
my bad, I thought the convention was (expected, actual). There's a lot of places that I've followed it. Will change it to (actual, expected)